rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2822,20 +3060,39 @@ extern void ppu_initialize()
 		compile_main = ppu_initialize(_main, true);
 	}
 
-	std::vector<lv2_prx*> prx_list;
+	std::vector<ppu_module*> module_list;
```

Comment:
Same here. I'm not going to check for any more standalone improvements.

Reply:
No, related.

Reply:
Standalone, not unrelated. I'm trying to think of how to merge the whole PR without too much risk.
By standalone I mean it can be submitted separately before the "main" work, e.g in a savestates part 1 PR or something.
Not that you'll do it anyway, but still I feel it would have really helped regtesting.